diorite-api/src/main/java/org/diorite/chat/placeholder/PlaceholderType.java

```diff
@@ -134,8 +134,8 @@
         PLUGIN.registerItem("description", BasePlugin::getDescription);
         PLUGIN.registerItem("website", BasePlugin::getWebsite);
         PLUGIN.registerItem("parent", BasePlugin::getName); //TODO
-        PLUGIN.registerItem("loaded", BasePlugin::getName); //TODO
-        PLUGIN.registerItem("loader", BasePlugin::getName); //TODO
+        PLUGIN.registerItem("loaded", BasePlugin::isInitialised);
+        PLUGIN.registerItem("loader", BasePlugin::getPluginLoader);
```

Comment:
To nie wygląda dobrze, bo nie wyświetli nic sensownego, już lepiej by było wyświetlić nazwę klasy plugin loadera, p -> p.getPluginLoader().getClass().getName()